packages/aws-sdk-client-mock-jest/test-d/types.ts

```diff
@@ -9,6 +9,9 @@ expect(1).toHaveReceivedCommand(PublishCommand);
 expect(mockClient(SNSClient)).toHaveReceivedCommand(PublishCommand);
 expectError(expect(mockClient(SNSClient)).toHaveReceivedCommand(String));
 
+expect(mockClient(SNSClient)).toHaveReceivedAnyCommand();
+expectError(expect(mockClient(SNSClient)).toHaveReceivedAnyCommand())
```

Comment:
This will likely fail. `expectError()` is verifying if providing wrong arguments to the function throws type errors, here there are no arguments. You can delete this line. The one above is okay.

Reply:
Thanks for clarifying. Removed it.